packages/core/src/services/chat.ts

```
@@ -1112,7 +1112,12 @@
                 proxyMode: Schema.const('on').required(),
                 proxyAddress: Schema.string().default('')
             }),
-            Schema.object({})
+            Schema.object({
+                proxyMode: Schema.const('off').required()
+            }),
+            Schema.object({
+                proxyMode: Schema.const('system')
+            }),
```

ESLint

[error] Delete `,`

(prettier/prettier)